.github/CODEOWNERS

```diff
@@ -134,6 +134,8 @@ ddtrace/contrib/internal/anthropic                      @DataDog/ml-observabilit
 ddtrace/contrib/anthropic                               @DataDog/ml-observability
 ddtrace/contrib/internal/google_generativeai            @DataDog/ml-observability
 ddtrace/contrib/google_generativeai                     @DataDog/ml-observability
+ddtrace/contrib/internal/vertexai                       @DataDog/ml-observability
+ddtrace/contrib/vertexai                                @DataDog/ml-observability
```

Comment:
Can we also add these here:
```
tests/contrib/vertexai/__init__.py                                      
tests/contrib/vertexai/conftest.py                                      
tests/contrib/vertexai/test_vertexai.py                                 
tests/contrib/vertexai/test_vertexai_patch.py                           
tests/contrib/vertexai/utils.py
```

Reply:
```suggestion
ddtrace/contrib/internal/vertexai                       @DataDog/ml-observability
ddtrace/contrib/vertexai                                @DataDog/ml-observability 
tests/contrib/vertexai                                  @DataDog/ml-observability 
```

Reply:
good catch! I added `tests/contrib/vertexai` to this section at the bottom (so it's near the other tests directory files).